CMakeLists.txt

```diff
@@ -79,3 +79,11 @@ target_compile_features(patomic_patomic PUBLIC c_std_90)
 if(NOT CMAKE_SKIP_INSTALL_RULES)
     include(cmake/util/InstallRules.cmake)
 endif()
+
+
+# ---- Setup Tests ----
+
+include(CTest)
+if(patomic_BUILD_TESTING)
```

Comment:
note: when I was reading this I falsely assumed I was supposed to use `patomic_BUILD_TESTING` to enable testing (instead of `BUILD_TESTING`) and this somehow screwed up ctest, making it think there were no tests at all - looking at the rest of the code doesn't seem to indicate any way in which not setting `BUILD_TESTING` could cause any problems, so I'm left with the assumption that `ctest` must be directly checking for `BUILD_TESTING` or something like that.

Reply:
Fixed.